public/controller/data_functions.php

```diff
@@ -278,8 +278,11 @@ function upload_homework($username, $semester, $course, $assignment_id, $homewor
 
     $settings_file = $user_path."/user_assignment_settings.json";
     if (!file_exists($settings_file)) {
-        $json = array("active_assignment"=>$upload_version);
-        file_put_contents($settings_file, json_encode($json));
+        date_default_timezone_set('America/New_York');
```

Comment:
Adding that line to ```public/index.php``` will affect any of the files under ```public/``` (like this file). The other found instances won't be affected (and should be left alone).